src/smt/optimization_solver.h

```diff
@@ -55,23 +55,43 @@ enum OptResult
   // the optimization loop finished and optimal
   OPT_OPTIMAL,
 
+  // the goal is unbounded, so it would be -inf or +inf
+  OPT_UNBOUNDED,
+
   // The last value is here as a preparation for future work
   // in which pproximate optimizations will be supported.
 
   // if the solver halted early and value is only approximate
   OPT_SAT_APPROX
 };
 
+/**
+ * The optimization objective,
```

Comment:
Can you ensure this uses the 80 char column limit better?

Reply:
Sure~

Reply:
Should be resolved